packages/nodes-base/nodes/Wait/Wait.node.ts

```diff
@@ -420,12 +420,17 @@ export class Wait extends Webhook {
 
 			waitAmount *= 1000;
 
-			waitTill = new Date(new Date().getTime() + waitAmount);
+			const original = new Date(new Date().getTime() + waitAmount);
+
+			waitTill = DateTime.fromISO(original.toISOString().slice(0, -1)).toUTC().toJSDate();
```

Comment:
Actually I just tested it now, and it looks real bad. It now takes into account the timezone, so it seems it startec picking up `America/New York` so instead of my workflows waiting 10 minutes, they now way 5 hours and 10 minutes, which is even worse 😓 

But to make things even more interesting, I changed the workflow's timezone, and it's also being completely ignored.

Reply:
Can you elaborate? This is how I tested:

- Selected 12:00 with workflow set at UTC-5 → Stored as 17:00 UTC in DB
- Selected +2h with workflow set at 10:40 UTC-5 → Stored as 17:40 UTC in DB

These are what I'd expect to be correct. What am I missing?